### ISSUE TITLE
Units of error in energy/forces during training

### ISSUE THREAD

**Issue by akashpg1996** [atomisticnet/aenet-PyTorch#1]
Hi aenet pytorch,
I have one doubt related to training process. During each epoch train.xaenet_pytorch.py prints error in training and test set. what are the units of energy and force errors ? are they per atom errors ?

Thanks
Akash

**Comment by victorccampos** [comment]
@jlopez141 Im facing the same issue. I search in the `py_aeio.py` but there is no comment regarding the units
